.goreleaser.yml

```diff
@@ -16,6 +17,19 @@ archives:
     windows: Windows
     386: i386
     amd64: x86_64
+release:
+  prerelease: auto
+brews:
+- github:
+    owner: vapor-ware
+    name: homebrew-formula
+  commit_author:
+    name: vio-bot
+    email: 'bot@vapor.io'
```

Comment:
I don't know what the vio-bot email is, so this may need to change.

Reply:
marco+viogh@vapor.io